wp/sections/work.tex

```diff
@@ -0,0 +1,7 @@
+\subsection{ktlint}
+\par
+Describe how does work ktlint (4-5 sentences)
+
+\subsection{diKTat}
+\par
+Describe how does work ktlint (5-6 sentences)
```

Comment:
I guess that we need to describe unique rules that are not checked and FIXED by other tools:
- identifier namings check (check that ktlint and detekt does not see it)
- commented code
- string concatenation
- chapters 5 and 6 - are unique
- e.t.c